### PR TITLE
Update ember-cli.

### PR DESCRIPTION
**Pull request by blimmer**
Also, resolve deprecation reported in #46.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,6 +63,7 @@
   "dependencies": {
     "bootstrap-sass": "^3.0.0",
     "broccoli-funnel": "^1.0.1",
+    "ember-cli-babel": "^5.1.7",
```
Comment (blimmer) [inline package.json:66]: This fixes #46

**Comment by blimmer** [comment]
I smoke tested this and it looks good.

**Comment by escobera** [comment]
:shipit:

**Comment by starlosgalaxy** [comment]
any idea when this will make it into the master branch?

**Comment by yankeeinlondon** [comment]
sorry, somehow missed this earlier

**Comment by yankeeinlondon** [comment]
published on npm as 0.5.6

**Pull request merged**